corehq/apps/app_manager/management/commands/helpers.py

```diff
@@ -26,6 +34,13 @@ def get_all_app_ids(domain=None, include_builds=False):
     ).all()}
 
 
+def get_deleted_app_ids(domain=None):
+    db = get_db_by_doc_type('Application')
+    return (list(iterate_doc_ids_in_domain_by_type(domain, 'Application-Deleted', database=db))
+        + list(iterate_doc_ids_in_domain_by_type(domain, 'LinkedApplication-Deleted', database=db))
+        + list(iterate_doc_ids_in_domain_by_type(domain, 'RemoteApp-Deleted', database=db)))
```

Comment:
if you're just going to coerce these to lists then you can just use `get_doc_ids_in_domain_by_type` which will return a list directly